docker/release/config/.vscode/cuda-quantum.code-profile

```diff
@@ -0,0 +1,5 @@
+{
+    "name": "CUDA Quantum",
+    "settings": "{\"settings\":\"{\\r\\n    \\\"workbench.colorTheme\\\": \\\"Default Light+\\\",\\r\\n    \\\"files.exclude\\\": {\\r\\n        \\\".*\\\": true,\\r\\n        \\\"*.o\\\": true\\r\\n        \\\"*.x\\\": true\\r\\n    }\\r\\n}\"}",
```

Comment:
Just wanted to confirm that you indeed wanted specify the `colorTheme` here instead of leaving it to the default value. (I only bring it up because it looks like if this may have been copy/pasted from something, and I wasn't sure if that part was intentional or not. I'm ok either way (though I'll be changing it to dark when I use it 😄 ).

Reply:
This profile is not loaded by default, but may be useful in some cases. You IDE is not going to turn white when you use CUDA Quantum, unless you import this.